parse/table_definitions_bsc/uniswap/UniswapV3Pool_event_Burn.json

```diff
@@ -0,0 +1,87 @@
+{
+    "parser": {
+        "type": "log",
+        "contract_address": "SELECT pool FROM ref('UniswapV3Factory_event_PoolCreated')",
```

Comment:
Can you submit a separate PR for the referred event (`UniswapV3Factory_event_PoolCreated`)? I see this in other chains but not for bsc. This needs to be submitted through a separate PR and merged first, then we can merge the current PR, so the tables can be built in order.

Reply:
PR for `UniswapV3Factory_event_PoolCreated` is [here](https://github.com/nansen-ai/evmchain-etl-table-definitions/pull/123).